apps/trading-e2e/src/support/step_definitions/deposits.step.ts

```diff
@@ -0,0 +1,75 @@
+import { And, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { EthereumWallet } from '../ethereum-wallet';
+import DepositsPage from '../pages/deposits-page';
+
+const depositsPage = new DepositsPage();
+const ethWallet = new EthereumWallet();
+
+const tBTC = '5cfa87844724df6069b94e4c8a6f03af21907d7bc251593d08e4251043ee9f7c';
```

Comment:
chore (blocking): we need to make this an environment variable, the contract addresses will not be consistent in capsule with the ones on ropsten

Reply:
can you set this in github secrets as `tBTC_contract` and we can call process.env.tBTC_contract in the test

Reply:
Does not need to be a secret just needs to be in the cypress env file, so you should need this one